Gemfile

```diff
@@ -114,6 +114,13 @@ group :development, :test do
   gem 'spring-commands-rspec'
 end
 
+group :development, :test do
```

Comment:
Should put these in the _existing_ `group :development, :test` section

Reply:
Done. I missed that section somehow...